README.rst

```diff
@@ -31,42 +31,61 @@
 
 |
 
-PyScaffold helps you setup a new Python project. Checkout out `this demo project`_, which was set up using Pyscaffold.
-In order to install PyScaffold, just pick your favourite installation method::
+PyScaffold is the tool of choice for bootstrapping high quality Python
+packages, ready to be shared on PyPI_ and installed via pip_.
+It is easy to use and encourages the adoption of the best tools and
+practices of the Python ecosystem, helping you (and your team, if your have one)
+to stay sane, happy and productive. The best part? It is stable and been used
+in the wild for over half a decade!
 
-    # Good old pip
+Checkout out `this demo project`_, which was set up using PyScaffold.
+If you are not convinced yet, have a look on these `reasons to use PyScaffold`_
+
+Quickstart
+==========
+
+Just pick your favourite installation method::
```

Comment:
Should we mention here already virtual environments in a way or at least give a reference? It seems that some people would just start typing those commands without creating a virtualenv or conda environment first.

Reply:
This is a tricky one... It is not a problem to install PyScaffold globally (and actually that is what I expect most of the people would do). The big problem is running the `pip install -e .` bit...

I will try to briefly mention about creating and activating virtual environments before the instruction for the editable install, in a way that it does not make the quick start too complex.